cluster/vsphere/util.sh

```diff
@@ -344,7 +344,7 @@ function kube-up {
 
   printf "Waiting for ${KUBE_MASTER} to become available..."
   until curl --insecure --user "${KUBE_USER}:${KUBE_PASSWORD}" --max-time 5 \
-          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta1/pods"; do
+          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta3/pods"; do
```

Comment:
BTW, we can't just make an API version constant, because semantics and path structure may be different for different versions. For instance, namespace should be in the path, in general, for v1beta3. It so happens that this path works, since some components need to list pods across all namespaces, but that capability will be restricted once authorization is enabled by default.

We should make a shell util function for checking whether the master is up.